README.md

```diff
@@ -4,98 +4,119 @@
+> **Learn more**: [Server Provider Documentation](./packages/openfeature-server-provider/README.md)
 
-### Accessing flags
+## Web Applications (Client-Side)
 
-The flag value API returns the Confidence assigned flag value or the passed in default value if no value was returned.
-The API supports dot notation, meaning that if the Confidence flag has a property `enabled` on the flag, you can access it directly.
+The following shows how to set up the client-side provider:
+
+```sh
+yarn add @openfeature/web-sdk @openfeature/core @spotify-confidence/openfeature-web-provider
+```
 
 ```ts
-const flag = await confidence.getFlag('tutorial-feature', {});
-if (flag.enabled) {
-  // ship it!
-}
-// or
-const enabled = await confidence.getFlag('tutorial-feature.enabled', false);
-if (enabled) {
-  // ship it!
+import { OpenFeature } from '@openfeature/web-sdk';
+import { createConfidenceWebProvider } from '@spotify-confidence/openfeature-web-provider';
+
+const provider = createConfidenceWebProvider({
+  clientSecret: 'your-client-secret',
+  fetchImplementation: window.fetch.bind(window),
+  timeout: 1000,
+});
+
+OpenFeature.setContext({
+  targetingKey: 'your-targeting-key',
+});
+
+try {
+  await OpenFeature.setProviderAndWait(provider);
+} (error) {
+  console.error('Failed to initialize Confidence provider:', error);
 }
-```
 
-> [!TIP]
-> If you are troubleshooting flag values, the flag evaluation API can be really useful since it returns a `FlagEvaluation` type that contain information about `variant`, `reason` and possible error details.
 
-```ts
-const flagEvaluation = await confidence.evaluateFlag('tutorial-feature', {});
+const client = OpenFeature.getClient();
+const isEnabled = client.getBooleanValue('feature.enabled', false);
 ```
 
-#### Caching
+> **Learn more**: [Web Provider Documentation](./packages/openfeature-web-provider/README.md)
 
-Flag evaluations are cached in memory on the Confidence instance with the evaluation context and flag name as a cache key.
-This is done to reduce network calls when evaluating multiple flags using the same context.
+---
 
-```ts
-const confidence = Confidence.create({...});
-const flag = confidence.getFlag('flag', {})
-// subsequent calls to getFlag will return the same value
-```
+# Direct SDK Usage
 
-If you need to always fetch the latest flag values (e.g., for testing, debugging or an other use case),
-you can bypass the cache by always get a fresh Confidence instance (and an empty cache):
+The vanilla sdk can be used in cases where you want direct access to the Confidence SDK, including event tracking and custom context management.
 
-```ts
-const confidence = Confidence.create({...});
-const flag = confidence.withContext({}).getFlag('flag', {})
-```
+> **Learn more**: [SDK Documentation](./packages/sdk/README.md)
+
+---
+
+# React Integration
```

Comment:
Maybe this should be another level in since it only applies to Direct SDK Usage.